### PR TITLE
chore: [IA-56] Upgrade react-native-cie

### PR DESCRIPTION
**Pull request by Undermaken**
## Short description
This PR upgrades `react-native-cie` to the last version
within version 1.0.7 `react-native-cie` includes two main changes proposed on the main [Android sdk repo ](https://github.com/italia/cieid-android-sdk)

1. italia/cieid-android-sdk#17
2. italia/cieid-android-sdk#19

these changes are included in this PR https://github.com/pagopa/io-cie-sdk/pull/34

1. IPZS (the CIE sdk Android mantainer) removed the certificates pinning: a security tech that restricts which certificates are considered valid for the communication client<->server
2. add a new error case: some Android devices (OPPO and Realme vendor) could not support a specific NFC feature required to read the CIE

### CODE REVIEW

File: locales/it/index.yml [hunk 1]
```diff
@@ -568,6 +568,7 @@ authentication:
     noDataTitle: Per usare IO è necessario autorizzare l'invio dei dati
     authToSendData: !include cie/auth_to_send_data.md
     nfc:
+      apduNotSupported: "Su questo dispositivo il produttore, per questa versione di sistema operativo, ha disabilitato la funzionalità necessaria per leggere la CIE mediante la tecnologia NFC"
```
Comment (thisisjp) [inline locales/it/index.yml:571]:
```suggestion
      apduNotSupported: "Il sistema operativo non supporta la lettura della tua carta di identità elettronica. Per maggiori informazioni, contatta il produttore del tuo dispositivo."
```

File: locales/en/index.yml [hunk 2]
```diff
@@ -560,6 +560,7 @@ authentication:
     noDataTitle: To use IO you must authorize the sending of data
     authToSendData: !include cie/auth_to_send_data.md
     nfc:
+      apduNotSupported: "On this device the manufacturer, for this version of the operating system, has disabled the functionality necessary to read the CIE using NFC technology"
```
Comment (thisisjp) [inline locales/en/index.yml:563]:
```suggestion
      apduNotSupported: "The operating system doesn't support the reading of your CIE. For further details, please contact your device manufacturer."
```

**Review by thisisjp — COMMENTED** [review]
@Undermaken I understand what you're trying to do here (there might be devices with NFC but not able to read CIE properly), but I'd rephrase the message in a less technical way :)

**Review by thisisjp — APPROVED** [review]

**Review by pietro909 — APPROVED** [review]
